packages/app/src/cli/services/context/identifiers-extensions.ts

```diff
@@ -161,9 +161,10 @@ function matchWebhooks(remoteSource: RemoteSource, extension: ExtensionInstance)
   const remoteVersionConfig = remoteSource.activeVersion?.config
   const remoteVersionConfigObj = remoteVersionConfig ? JSON.parse(remoteVersionConfig) : {}
   const localConfig = extension.configuration as unknown as SingleWebhookSubscriptionType
+  const remoteUri: string = remoteVersionConfigObj.uri || ''
   return (
     remoteVersionConfigObj.topic === localConfig.topic &&
-    remoteVersionConfigObj.uri === localConfig.uri &&
+    (remoteUri === localConfig.uri || remoteUri.endsWith(localConfig.uri)) &&
```

Comment:
If a user has a two webhooks with same topic, same filter but different (but similar) URLs, this could trigger a false positive.
Example:
```json
{
  topic: "products/create",
  filter: "something",
  uri: "/products/webhooks"
}
```
vs 
```json
{
  topic: "products/create",
  filter: "something",
  uri: "/webhooks"
}
```
This could both return true for a full url like: `https://<domain>/products/webhooks

Reply:
The second condition already covers the first one, so we can remove it.

```suggestion
    remoteUri.endsWith(localConfig.uri) &&
```